client/src/components/publications/publications.css

```diff
@@ -29,3 +29,7 @@
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
 }
+
+.placeholder-text::placeholder {
+  color: #A9A9A9
+}
```

Comment:
Please remove if unused

Reply:
It's being used now that we're using classname styling. See comment above https://github.com/Researchify/Researchify/pull/150#discussion_r686726239